packages/app/src/debug/DebugPageDetails.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="flex flex-col w-full px-24px justify-center items-center align-middle"
+    :class="{'flex-grow': ['PASSED', 'OVERLIMIT'].includes(status)}"
+  >
+    <DebugCancelledAlert
+      v-if="status === 'CANCELLED'"
+      :total-specs="totalSpecs"
+      :total-skipped-specs="totalSkippedSpecs"
+      :canceled-at="canceledAt"
+      :canceled-by-full-name="canceledByFullName"
+      :canceled-by-email="canceledByEmail"
+    />
+    <DebugPassed v-else-if="status === 'PASSED'" />
+    <DebugErrored
+      v-else-if="status === 'ERRORED'"
+      :errors="errors"
+      :total-specs="totalSpecs"
+      :total-skipped-specs="totalSkippedSpecs"
+    />
+    <DebugNoTests v-else-if="status === 'NOTESTS'" />
+    <DebugTimedout
+      v-else-if="status === 'TIMEDOUT'"
+      :total-specs="totalSpecs"
```

Comment:
I think we can just pass `props.specs.length`, no need for the computed, since we are not deriving anything. `props` are reactive out of the box iirc (like React).